### PR TITLE
 Update estimates button on location search must make user aware of changes

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description 
On the search by location, the "Update tuition and housing estimates" impacts data in the search results, not the map - users have no indication that changes are happening in the data. Users must have an indicator of how changes are impacting their search results.

## Original issue(s)
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/29476

## Testing done
local

## Screenshots

Screen Shot 2021-09-02 at 11 38 27 AM
Screen Shot 2021-09-02 at 11 38 43 AM

## Acceptance criteria
- [x] The following note is displayed above the "Update estimates" button of the "Update tuition and housing estimates" component:

Note: Changing these settings modifies the tuition and housing benefits shown on the search cards.

- [x] The note can be navigated to via the keyboard
a. The user can tab from the "Will you be taking any classes in person?" radio buttons to the note.
b. The user can tab to the "Update estimates" button.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx [hunk 1]
```diff
@@ -93,6 +93,11 @@ export function TuitionAndHousingEstimates({
           setOnlineClasses(e.target.value);
         }}
       />
+      {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+      <div tabIndex="0" className="vads-u-padding-top--2">
```
Comment (timwright12) [inline src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx:97]: @ksantiagoBAH what are we using this tabindex for? It doesn't seem like it's something that needs to be focusable. If it needs programmatic focus (like an expanding box or something), we would set that value to `-1` so it doesn't interfere with normal keyboard usage

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx (hunk 1 above)
Comment (ksantiagoBAH) [reply src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx:97]: It was requested in a 508 defect by @noahgelman  for screen reader users to announce that changes that will occur.

Comment (timwright12) [reply src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx:97]: This wouldn't force an announcement, just interrupt the tab order. There are some best practices for things like this here: https://www.w3.org/WAI/tutorials/forms/instructions/ namely `aria-labelledby` or `aria-describedby`

Comment (noahgelman) [reply src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx:97]: @timwright12 The content that they are adding tabindex to is not information related to any specific field. It is a general note that describes an important functionality of the page that we want screen reader users to not miss as they would be tabbing through the field. It's more akin to a warning or alert which a tabindex is sometimes added to make sure it's tabbed to.

Comment (timwright12) [reply src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx:97]: @noahgelman I'll defer to your expertise, but wouldn't this be more like an aria-describedby on the submit button in that were the case or maybe set to the form itself?

```
<form aria-describedby="thenote">
  <p id="thenote"></p>
```

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by timwright12 — APPROVED** [review]

File: src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx (hunk 1 above)
Comment (noahgelman) [reply src/applications/gi-sandbox/containers/TuitionAndHousingEstimates.jsx:97]: @timwright12 I think putting it on the submit button should be fine

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**